examples/v2/chat2.nim

```diff
@@ -238,7 +238,7 @@ proc publish(c: Chat, line: string) =
   else:
     # No payload encoding/encryption from Waku
     var message = WakuMessage(payload: chat2pb.buffer,
-      contentTopic: c.contentTopic, version: 0)
+      contentTopic: c.contentTopic, version: 0, timestamp: Timestamp(time*1000*1000*1000))
```

Comment:
You can use getNanosecondTime() for the conversion (and omit Timestamp type specification):

```suggestion
      contentTopic: c.contentTopic, version: 0, timestamp: getNanosecondTime(time))
```